src/f32-gemm/MrxLmulv-rvv.c.in

```diff
@@ -0,0 +1,170 @@
+// Copyright 2023 SiFive, Inc.
+//
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree.
+
+$assert DATATYPE in ["F32"]
```

Comment:
looks like this code would work for fp16 if we passed it as a datatype.

Reply:
Yes, it can be changed to fp16 with small modifications.